### PR TITLE
Refine bulgarian translations

### PR DESCRIPTION
**Pull request by kachar**
The invite more people prompt was partially translated as there were missing keys compared to the `master` version of `lang/main.json` file

With this PR we update the keys and the respective translations to improve the UX.
Multiple typos were fixed to comply with bulgarian grammar rules.

- [x] Signed https://jitsi.org/icla

### CODE REVIEW

File: lang/main-bg.json [hunk 1]
```diff
@@ -773,7 +785,7 @@
         "header": "Място за помощ"
     },
     "lonelyMeetingExperience": {
-        "youAreAlone": "Вие сте сами в срещата",
+        "youAreAlone": "Вие сте сам в срещата",
```
Comment (damencho) [inline lang/main-bg.json:788]: Do you think we can re-phrase this so it's gender-neutral?

Comment (damencho) [reply lang/main-bg.json:788]: I haven't checked the other updates for gender-nutral strings, but this was one thing I was trying to achieve when updating the translations, the last time I did it? :)

**Comment by damencho** [comment]
Thank you for your contribution!!!

Jenkins, test this please.

File: lang/main-bg.json (hunk 1 above)
Comment (kachar) [reply lang/main-bg.json:788]: @damencho Thanks for the update! We'll do an iteration to make sure that the strings are gender neutral.

**Review by damencho — APPROVED** [review]

**Pull request merged**